Scripts/Game.gd

```diff
@@ -0,0 +1,19 @@
+extends Node2D
+
+export var path_to_load  = "res://Scenes/World.tscn"
+
+func _ready():
+	add_scene(path_to_load)
+	
+func add_scene(path = "res://Scenes/World.tscn"):
+	var scene = load(path)
+	if scene == null:
+		# handle this by going to the default path?
```

Comment:
I guess is fine to crash if the scene does not exist

Reply:
I think I'll load the title screen, doesn't matter much though.